### PR TITLE
Set OPENSSL_ENGINES for Windows

### PR DESCRIPTION
**Pull request by bernd-edlinger**

### CODE REVIEW

File: Configurations/windows-makefile.tmpl [hunk 1]
```diff
@@ -220,6 +220,7 @@ test: tests
 	set BLDTOP=$(BLDDIR)
 	set RESULT_D=$(BLDDIR)\test\test-runs
 	set PERL=$(PERL)
+	set OPENSSL_ENGINES=$(BLDDIR)\engines
```
Comment (levitte) [inline Configurations/windows-makefile.tmpl]: Have you tested that this works? I think that it needs to be an absolute directory...

**Comment by bernd-edlinger** [comment]
Oops, wrong version checked in, OPENSSL_ENGINES needs to be an absolute path.

**Comment by bernd-edlinger** [comment]
I need to use MAKEDIR not BLDDIR, sorry.

File: Configurations/windows-makefile.tmpl (hunk 1 above)
Comment (bernd-edlinger) [reply Configurations/windows-makefile.tmpl]: oops....

**Comment by bernd-edlinger** [comment]
fixed, no idea how git could pick up an old version...

**Comment by bernd-edlinger** [comment]
Merged.  Thanks!

**Pull request closed without merging**